src/pages/Merchant/LoginPage/LoginPage.js

```diff
@@ -37,9 +41,18 @@ function LoginPage() {
 
   const handleFormSubmit = (event) => {
     event.preventDefault();
-    if (!validateForm()) {
-      return;
-    }
+    if (!validateForm()) return;
+
+    const loginDetails = {
+      mobile: "",
```

Comment:
if mobile field is not needed in the api then just dont put it in this object just eliminate it

Reply:
it is in body of api in postman but do we really need this

Reply:
??